lib/src/components/NxTable/NxTable.scss

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../base-styles/nx-globals';
+@import '../../base-styles/nx-tables';
```

Comment:
These imports will result in the `nx-table` styles being present twice in the output css.  Given the way the RSC styles are intended to work, I think you can simply get rid of this file

Reply:
If I do that, then I'll have to build this scss file in nxrm when I use these components which is fine. Outside the scope of this PR, but we're inconsistent in when I need to do that and when not. I'll remove this.

Reply:
> Outside the scope of this PR, but we're inconsistent in when I need to do that and when not.

Where are we inconsistent about it?  We do have consistent rules about it as described in the README.

The two use cases that we aim to best support are:
1. Those that are having their styles built automatically based on the imports they are using.  It is possible that this currently doesn't as efficiently as intended, due to the way side-effect imports work, importing `index.ts` might be bringing in all component styles even those for components that aren't being used.
2. Those that just want to import all RSC styles as a precompiled css file, which we generate during the build.

If you are doing something in between either of those two options, then you'll have to work things out manually to some extent, but generally the idea would be that you (the consuming application) need to import all of the base styles and then the component-specific styles for all components that you are using.

Reply:
Right, but I'm doing neither right now. I'm importing/exporting individual components which sometimes include their own styles and sometimes not. As you say though, we're not targeting that as a best practice so it's probably fine. It's been a little while since I've looked at it, I may soon be able to just import all the RSC styles now that I've started using an override file.

Reply:
> sometimes include their own styles and sometimes not.

Please elaborate on this.  Outside of the `base-styles`, I don't know of any styles required by components which the components themselves fail to pull in.  You should be pulling in all of the `base-styles` if you are using any part of RSC; that's  one of the assumptions we make.

Reply:
The NxButton component doesn't pull in any styles by just importing it. I need to look at just importing all the styles by default now to see if it causes any problems as it's been a while since I tried it.

Reply:
> The NxButton component doesn't pull in any styles by just importing it. I need to look at just importing all the styles by default now to see if it causes any problems as it's been a while since I tried it.

Note that the styles that `NxButton` relies on, `nx-btn`, have their own page in the gallery.  They are considered base styles.  This is the same situation here with `NxTable` and `nx-table`.